OasisDEX/oasis-borrow
# Add PositionList for Earn overview page.

# [Create generic table component](https://app.shortcut.com/oazo-apps/story/3781/create-generic-table-component)

<please insert a shortcut link above>
  
## Changes 👷‍♀️
  <Please add short list of changes>

- Add PositionList and a couple of stories.
  
## How to test 🧪
  <Please explain how to test your changes>

- Check code.
- Check storybook here: https://earn-table-storybook.surge.sh/?path=/story/positionlist--positions
